assets/animations.js

```diff
@@ -25,4 +33,7 @@ function initializeScrollAnimationTrigger(rootEl = document) {
 
 window.addEventListener('DOMContentLoaded', () => initializeScrollAnimationTrigger());
 
-document.addEventListener('shopify:section:load', (event) => initializeScrollAnimationTrigger(event.target));
+if (Shopify.designMode) {
+  document.addEventListener('shopify:section:load', (event) => initializeScrollAnimationTrigger(event, event.target));
+  document.addEventListener('shopify:section:reorder', (event) => initializeScrollAnimationTrigger(event));
```

Comment:
If you don't pass in an element to `initializeScrollAnimationTrigger` it will create a new intersection observer for the entire document, which already has one from line 34. Can we just create the new observer for the section(s) that was reordered?

Reply:
I had it for the specific section in the first place but it worked in a way that only the re ordered section was not getting animated while every other section was ([video](https://screenshot.click/17-02-bkbxc-6bz2f.mp4)). 

What I did just now was adding an early return within my if statement. So that if the event triggering the initialization is coming from the editor (`(re)load` or `reorder`) so we're not observing more element as it's unnecessary since the classes applied prevent the animation from happening anyway. 

https://github.com/Shopify/dawn/pull/2502/commits/e89ed0294309e4ece9879885f7dddd0c19ff0f55